### ISSUE TITLE
[Bug Report] Adding color to stepper doesn't do anything

### ISSUE THREAD

**Issue by CindyConway** [vuetifyjs/vuetify#4373]
### Versions and Environment
**Vuetify:** 1.0.19
**Vue:** 2.5.16
**Browsers:** Chrome 67.0.3396.87
**OS:** Mac OS 10.13.4

### Steps to reproduce
Add a stepper to a page and set the color property like this:

```html
             <v-stepper :value="1" alt-labels>
              <v-stepper-header>

                <v-stepper-step 
                  color="my-class"
                  :step="1" 
                  >test</v-stepper-step>
                <v-divider></v-divider>
                <v-stepper-step 
                  color="my-class"
                  :step="2" 
                  >test</v-stepper-step>
                <v-divider></v-divider>

              </v-stepper-header>
            </v-stepper>
````

### Expected Behavior
The color of the steps are changed

### Actual Behavior
The color of the step doesn't change because the value passed in to color is not added as a class or style. All that is added to the element is: "color='some-class'", which, of course, does nothing.

### Reproduction Link
[https://codepen.io/CindyConway/pen/gKzZOQ/](https://codepen.io/CindyConway/pen/gKzZOQ/)

### Other comments
Here is the generated code

```html
<div data-v-8ade141c="" class="stepper stepper stepper--alt-labels">
  <div data-v-8ade141c="" class="stepper__header">
    <div data-v-8ade141c="" class="stepper__step stepper__step--active" color="my-class">
       <span class="stepper__step__step primary">1</span>
       <div class="stepper__label">Sign Agreement</div>
     </div> 
      <hr data-v-8ade141c="" class="divider">
      <div data-v-8ade141c="" class="stepper__step stepper__step--inactive" color="my-class"> . 
         <span class="stepper__step__step">2</span>
          <div class="stepper__label">Pay Invoice</div>
      </div> 
       <hr data-v-8ade141c="" class="divider">
      <div data-v-8ade141c="" class="stepper__step stepper__step--inactive" color="my-class">
           <span class="stepper__step__step">3</span>
           <div class="stepper__label">View Report</div>
       </div> <!---->
    </div>
</div>
```

**Comment by KaelWD** [comment]
Duplicate of #3276

**Comment by CindyConway** [comment]
For anyone who comes across this, #3276 is, in fact, for the same issue, but that issue just says that the fix isn't in 1.0.17. I am on 1.0.19, the latest release at the time, so I figured there was still a problem.

As of the writing of this comment, the fix for stepper color is still  only in dev branch (1.1.0-rc.2) which I can't use because I don't want to put a release candidate in production, but I'll keep an eye out for the next release.
